src/cli.rs

```diff
@@ -545,19 +545,24 @@ async fn run_deploy(
 
         if deploy_data.merged_settings.interactive_sudo.unwrap_or(false) {
             warn!("Interactive sudo is enabled! Using a sudo password is less secure than correctly configured SSH keys.\nPlease use keys in production environments.");
+            
+            if deploy_data.merged_settings.sudo.is_some() {
+                warn!("Custom sudo commands should be configured to accept password input from stdin when using the 'interactive sudo' option. Deployment may fail if the custom command ignores stdin.");
+            } else {
+                // this configures sudo to hide the password prompt and accept input from stdin
+                if let Some(original) = deploy_defs.sudo.clone() {
+                    if !original.contains("-S") {
+                        deploy_defs.sudo = Some(format!("{} -S -p \"\"", original));
```

Comment:
This line implicitly implies that the provided `deploy_defs.sudo` supports both `-S` and `-p` options. IMO, it's better to leave all options to be provided as a custom `sudo` instead of changing what is provided by the flake/command line option, i.e. completely avoid modifying `deploy_defs.sudo` if it's not a `None`
What do you think?